workflow/scripts/generate-lineage-variant-table.py

```diff
@@ -128,70 +160,75 @@ def has_numbers(inputString):
 sorterIndex = dict(zip(sorter, range(len(sorter))))
 variants_df["Features_Rank"] = variants_df["Features"].map(sorterIndex)
 
-# define categories for sorting
-variants_df.loc[
-    (variants_df[top5_lineages[1]] == "x") & (variants_df["Probability"] >= 0.95),
-    "Order",
-] = 0
-variants_df.loc[
-    (variants_df[top5_lineages[1]] == "x") & (variants_df["Probability"] <= 0.05),
-    "Order",
-] = 1
-variants_df.loc[
-    (variants_df[top5_lineages[1]] != "x") & (variants_df["Probability"] >= 0.95),
-    "Order",
-] = 2
-variants_df.loc[
-    (variants_df[top5_lineages[1]] == "x")
-    & ((variants_df["Probability"] > 0.05) & (variants_df["Probability"] < 0.95)),
-    "Order",
-] = 3
-variants_df.loc[
-    (variants_df[top5_lineages[1]] != "x") & (variants_df["Probability"] <= 0.05),
-    "Order",
-] = 4
-variants_df.loc[
-    (variants_df[top5_lineages[1]] != "x")
-    & ((variants_df["Probability"] > 0.05) & (variants_df["Probability"] < 0.95)),
-    "Order",
-] = 5
-
-top5_lineages_row_df = pd.DataFrame(
-    {"Mutations": "Lineage", **{x: x for x in top5_lineages[1:6]}}, index=[0]
-)
-
-# sort final DF
-variants_df["Prob X VAF"].replace([0, 0.0], np.NaN, inplace=True)
-variants_df.sort_values(
-    by=["Order", "Features_Rank", "Position"],
-    ascending=[True, True, True],
-    na_position="last",
-    inplace=True,
+# row for lineage name after renaming columns (column names can't be formatted)
+lineages_row_df = pd.DataFrame(
+    {
+        "Mutations": "Lineage",
+        **{x: x for x in list(lineage_df.columns) if x != "Mutations"},
+    },
+    index=[0],
 )
 
 # concat row with Jaccard coefficient, drop unneccesary columns, sort with Jaccard coefficient, round
-variants_df = pd.concat([jaccard_row, variants_df]).reset_index(drop=True)
-variants_df = pd.concat([top5_lineages_row_df, variants_df]).reset_index(drop=True)
-variants_df = variants_df[
-    ["Mutations", "Probability", "Frequency", *top5_lineages[1:6]]
-]
-variants_df = variants_df.round({"Probability": 5, "Frequency": 5})
+variants_df.reset_index(inplace=True)
+variants_df = pd.concat([jaccard_row, variants_df])
+variants_df = pd.concat([lineages_row_df, variants_df])
+
+
+variants_df = variants_df.round({"Probability": 2, "Frequency": 2})
 variants_df.set_index("Mutations", inplace=True)
 variants_df.sort_values(
     by="Similarity", axis=1, na_position="first", ascending=False, inplace=True
 )
-# rename top 5 hits
+# rename hits ascending
 variants_df.rename(
     columns={
         x: y
         for x, y in zip(
-            list(variants_df.columns)[2:],
-            ["Highest similarity", "2nd", "3rd", "4th", "5th"],
+            list(variants_df.columns)[8:],
+            rename_enumeration(len(list(variants_df.columns)[8:])),
```

Comment:
Would it be a good idea to make this adjustable via the params? No really important, just an idea.

Reply:
do you mean how many top lineages adjustable?

Reply:
Yes, having the number of lineages configurable seems like a good idea. Currently the names are kind of hardcoded though. As this will become much easier with datavzrd (since the special column names aren't needed then), I suggest to handle this in a separate PR once this has been migrated to datavzrd.

Reply:
So I'm going to adress this in an issue for now

Reply:
Sound good to me 👍